<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceGroupBox</class>
 <widget class="QGroupBox" name="SourceGroupBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Image Source</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="sourceTypeComboBox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400; text-decoration: underline;&quot;&gt;IMAGE SOURCE&lt;/span&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; types:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Hidra&lt;span style=&quot; font-weight:400;&quot;&gt; - image from a Hidra server, e.g. &lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;Pilatus, Perkin Elmer&lt;/span&gt;&lt;/p&gt;&lt;p&gt;HTTP response &lt;span style=&quot; font-weight:400;&quot;&gt;- image from an http response, e.g. &lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;Eiger&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Tango Attribute&lt;span style=&quot; font-weight:400;&quot;&gt; - image from a tango attribute, e.g. &lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;Lambda, PCO &lt;/span&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;or &lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;LimaCCDs&lt;/span&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; detectors&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Tango Events - &lt;span style=&quot; font-weight:400;&quot;&gt;image from a tango attribute events, e.g. &lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;LimaCCDs&lt;/span&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; detectors&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Tango File &lt;span style=&quot; font-weight:400;&quot;&gt;- image defined by file and directory tango attributes, e.g. &lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;Pilatus&lt;/span&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; w/o Hidra&lt;/span&gt;&lt;/p&gt;&lt;p&gt;ZMQ Stream &lt;span style=&quot; font-weight:400;&quot;&gt;- image from a simple ZMQ server, e.g. &lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;lavuezmqstreamfromtango&lt;/span&gt;&lt;span style=&quot; font-weight:400;&quot;&gt; SERVER&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Nexus File &lt;span style=&quot; font-weight:400;&quot;&gt;- image from a Nexus/Hdf5 file, e.g. written with &lt;/span&gt;&lt;span style=&quot; font-weight:400; font-style:italic;&quot;&gt;SWMR&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Test&lt;span style=&quot; font-weight:400;&quot;&gt; - random test image&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">font: bold;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>start/stop reading images</string>
       </property>
       <property name="text">
        <string>&amp;Start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cStatusLineEdit">
       <property name="styleSheet">
        <string notr="true">color: blue; background-color: yellow;</string>
       </property>
       <property name="text">
        <string>Not connected</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="cStatusLabel">
       <property name="toolTip">
        <string>connection status and via which port the zmq security stream is being emitted</string>
       </property>
       <property name="text">
        <string>Status:</string>
       </property>
       <property name="buddy">
        <cstring>cStatusLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="sourceTypeLabel">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;IMAGE SOURCE&lt;/span&gt; types:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Hidra&lt;/span&gt; - image from a Hidra server, e.g. &lt;span style=&quot; font-style:italic;&quot;&gt;Pilatus, Perkin Elmer&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;HTTP response&lt;/span&gt; - image from an http response, e.g. &lt;span style=&quot; font-style:italic;&quot;&gt;Eiger&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Tango Events&lt;/span&gt; - image from a tango attribute events, e.g. &lt;span style=&quot; font-style:italic;&quot;&gt;LimaCCDs&lt;/span&gt; detectors&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Tango Attribute &lt;/span&gt;- image from a tango attribute, e.g. &lt;span style=&quot; font-style:italic;&quot;&gt;Lambda, PCO &lt;/span&gt;or &lt;span style=&quot; font-style:italic;&quot;&gt;Lima&lt;/span&gt; detectors&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Tango File&lt;/span&gt; - image defined by file and directory tango attributes, e.g. &lt;span style=&quot; font-style:italic;&quot;&gt;Pilatus&lt;/span&gt; w/o Hidra&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ZMQ Stream&lt;/span&gt; - image from a simple ZMQ server, e.g. &lt;span style=&quot; font-style:italic;&quot;&gt;lavuezmqstreamfromtango&lt;/span&gt; SERVER&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Nexus File &lt;/span&gt;- image from a Nexus/Hdf5 file, e.g. written with &lt;span style=&quot; font-style:italic;&quot;&gt;SWMR&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Test&lt;/span&gt; - random test image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Source:</string>
       </property>
       <property name="buddy">
        <cstring>sourceTypeComboBox</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
